add-sailthru-paths.js

```diff
@@ -33,6 +33,10 @@ function encodePath(path) {
     .join("*")
 }
 
+function isNotSailthruPattern(pattern) {
+  return !/^NOT \/click\/\*/.test(pattern)
+}
```

Comment:
This is so we can add manually Sailthru encoded URLs and not have them encoded again.